index.js

```diff
@@ -125,16 +127,17 @@ module.exports = function(S) { // Always pass in the ServerlessPlugin Class
                     table = config && config.table || {},
                     rootPath = S.getProject().getRootPath(),
                     tablesPath = rootPath + '/' + (table.dir || 'dynamodb'),
+                    dbPort = config && config.start && config.start.port || 8000,
+                    dbOptions = { region: 'localhost', endpoint: 'http://localhost:' + dbPort },
+                    dynamodb = { raw: new AWS.DynamoDB(dbOptions), doc: new AWS.DynamoDB.DocumentClient(dbOptions) },
```

Comment:
Will it be possible to use this library there?
https://github.com/99xt/serverless-dynamodb-client